.github/workflows/mobile-deploy-auto.yml

```diff
@@ -3,7 +3,7 @@ name: Mobile Auto Deploy
 on:
   pull_request:
     types: [closed]
-    branches: [main, dev]
+    branches: [staging]
```

Comment:
@remicolin changed the auto deploy flow to kick off after merge to staging. that makes the most sense especially given our flow of manually promoting builds for release